editor/sidebar/last-revision/index.js

```diff
@@ -57,7 +57,8 @@ class LastRevision extends Component {
 		}
 		this.setState( { loading: true } );
 		const postIdToLoad = this.props.postId;
-		this.fetchRevisionsRequest = new wp.api.collections.PostRevisions( {}, { parent: postIdToLoad } ).fetch()
+		const Collection = wp.api.getPostTypeRevisionsCollection( this.props.postType || 'post' );
```

Comment:
I think I'll just remove the `|| 'post'` part altogether. If the collection doesn't exist, then it should be due to revisions not being supported, and the component should render nothing.